templates/register.html

```diff
@@ -77,12 +77,12 @@ <h2 class="text-center mb-4">New user</h2>
             <div class="row justify-content-center">
                 <div class="col-sm-4" id="supervisor_select" style="display: none;">
                     <br>
-                    <label for="supervisor">Supervisor</label>
-                    <select class="form-select" id="supervisor" name="supervisor">
-                        {% for supervisor in supervisors %}
-                            <option value="{{ supervisor.id }}">{{ supervisor.first_name }} {{ supervisor.name }}</option>
-                        {% endfor %}
-                    </select>
+                    <div class="form-group">
+                        <label for="supervisor">Supervisor</label>
+                        {% with page='register' %}
```

Comment:
Conditions :
- supervisor select not disabled
- researcher not defined yet => empty supervisor list